cmd/frontend/internal/app/debug_test.go

```
@@ -109,22 +108,18 @@ func Test_prometheusValidator(t *testing.T) {
 }
 
 func TestGrafanaLicensing(t *testing.T) {
-	if testing.Short() {
-		t.Skip()
-	}
-
 	t.Run("licensed requests succeed", func(t *testing.T) {
-		database.Mocks.Users.GetByCurrentAuthUser = func(ctx context.Context) (*types.User, error) {
-			return &types.User{ID: 1, SiteAdmin: true}, nil
-		}
-		defer func() { database.Mocks.Users.GetByCurrentAuthUser = nil }()
+		users := database.NewStrictMockUserStore()
```

TIL about strict mock... but not remembering seeing them being used before, should we all use strict mocks or based on what criteria?

---

[They're new](https://github.com/derision-test/go-mockgen/pull/16)! The only difference is that any unmocked calls will panic rather than returning zero values. I personally find it nicer to get a panic message that says "unexpected call to unmocked method" rather than just getting an unexplained nil pointer error somewhere down the line.